### PR TITLE
Feat: changes for 2.4.0 release

### PR DESCRIPTION
**Pull request by erezz-mov-ai**
#32 

- [x] Make sure you are opening from a **topic/feature/bugfix branch**
- [x] Ensure that the PR title represents the desired changes
- [x] Ensure that the PR description detail the desired changes
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
'- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue'

[^note]:
    Put an `x` into the [ ] to show you have filled the information.
    The template comes from https://github.com/MOV-AI/.github/blob/master/.github/pull_request_template.md
    You can override it by creating .github/pull_request_template.md  in your own repository

### CODE REVIEW

**Review by engdorm — APPROVED** [review]

**Comment by engdorm** [comment]
@AlexFernandes-MOVAI added the missing reqs 
please review

File: tox.ini [hunk 1]
```diff
@@ -4,6 +4,10 @@ envlist = py38
 
 [testenv]
 # the requirements to be installed in the environment in which the tests
+setenv =
```
Comment (diasdm) [inline tox.ini:7]: Let's discuss this is today's meeting

**Review by diasdm — CHANGES_REQUESTED** [review]

File: tox.ini (hunk 1 above)
Comment (AlexFernandes-MOVAI) [reply tox.ini:7]: As agreed, I have put integration repo instead of experimental

**Comment by AlexFernandes-MOVAI** [comment]
need gd-node to be released first:
`ERROR: Could not find a version that satisfies the requirement gd-node==1.0.1.0 (from flow-initiator) (from versions: 1.0.0.1, 1.0.0.2, 1.0.0.3, 1.0.0.4, 1.0.0.5, 1.0.0.6, 1.0.0.7, 1.0.0.8, 1.0.0.9)
ERROR: No matching distribution found for gd-node==1.0.1.`

File: setup.py [hunk 2]
```diff
@@ -6,13 +6,13 @@
...
 ]
 
 
 setuptools.setup(
     name="flow-initiator",
-    version="1.0.1-10",
+    version="2.4.0-0",
     author="Backend team",
     author_email="backend@mov.ai",
     description="Dummy description",
```
Comment (AlexFernandes-MOVAI) [inline setup.py:18]: Please change `dummy description`

File: .bumpversion.cfg [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.0.1-10
+current_version = 2.4.0-0
 commit = True
 message = 
 	[SKIP] Automatic version bump {current_version} -> {new_version}
```
Comment (AlexFernandes-MOVAI) [inline .bumpversion.cfg:5]: please change [SKIP] by [skip actions]

**Review by AlexFernandes-MOVAI — CHANGES_REQUESTED** [review]
small changes requested

**Pull request merged**